### PR TITLE
Add Rails release process for Foreman version

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: procedures/foreman/branch.md.erb [hunk 1]
```diff
@@ -20,6 +20,10 @@
 * [ ] Add version <%= release %> to jobs in axes and/or combination filters, remove old ones (keep three)
 * [ ] Clone Debian nightly repos to <%= release %> using [copy/freight instructions](https://projects.theforeman.org/projects/foreman/wiki/Debian_Packaging#Project-sources)
 * [ ] Edit [foreman-packaging's PR template](https://github.com/theforeman/foreman-packaging/blob/master/.github/PULL_REQUEST_TEMPLATE.md) to add <%= release %> and remove the old release
+* [ ] Include Rails repository
+  * [ ] Add tfm-ror52 tag entires to Foreman tool_belt config to copy from main tfm-ror52 tag
```
Comment (ekohl) [inline procedures/foreman/branch.md.erb]: entires -> entries

**Comment by ehelms** [comment]
I'd link to previous examples, but we have none yet of this strategy. I suppose we can go through the process and then try to update the wording. Speaking of, we should get a 1.21 config up.... @ekohl  do you want to handle that or shall I?

**Comment by ekohl** [comment]
That sounds good. Feel free to put one up.

**Comment by ehelms** [comment]
@ekohl Updated

File: procedures/foreman/branch.md.erb [hunk 2]
```diff
@@ -20,6 +20,11 @@
 * [ ] Add version <%= release %> to jobs in axes and/or combination filters, remove old ones (keep three)
 * [ ] Clone Debian nightly repos to <%= release %> using [copy/freight instructions](https://projects.theforeman.org/projects/foreman/wiki/Debian_Packaging#Project-sources)
 * [ ] Edit [foreman-packaging's PR template](https://github.com/theforeman/foreman-packaging/blob/master/.github/PULL_REQUEST_TEMPLATE.md) to add <%= release %> and remove the old release
+* [ ] Include Rails repository
+  * [ ] Add tfm-ror52 tag entries to Foreman tool_belt config to copy from main tfm-ror52 tag
```
Comment (ekohl) [inline procedures/foreman/branch.md.erb]: Should this be `Add foreman-rails-<%= release %>` tag entries? Or just explicitly `foreman-rails-<%= release %>-rhel7`

**Comment by ehelms** [comment]
Updated

**Pull request merged**